### DISCUSSION TITLE
Tutorial for people just using other people's repositories?

### DISCUSSION THREAD

**Discussion by fugueink** [skills/.github#214] [General]
I've used GitHub for years, but very poorly. I don't understand the language, how to get help, any of it.

Everything I look at in the way of help is written for developers, people making repositories on GitHub. Doesn't someone, somewhere, have a tutorial on how to use GitHub as just an end user of other people's repositories?

I want to use Open Source software, but . . . I have to keep giving up because I can't climb through the code and I cannot even figure out where to ask for help on individual packages on GitHub. I've upset a lot of people by trying to. At best, I get ignored and have to give up on using whatever it is. Unless the program has something I can just drag into my Applications folder (I'm on an older Mac), I'm out of luck.

Thanks for any help I can get here. Please don't yell at me if this isn't the right place to ask.

**Comment by SimonVerhoeven** [comment]
I think therein lies the quandary, and indeed this section isn't really for your question.
These skills, and most repositories are used to store the source code for the end files you are looking for (and hence the dicussion/help pages are aimed at helping people to build on that code/report bugs in it). Albeit some repositories do indeed include the executables you are looking for.
